### PR TITLE
Updating Apollos Assets

### LINKED ISSUE #380: Update Apollos Assets
## Acceptance Criteria

- [x] Apollos should display the new seasonal splash screen
- [x] Apollos on web should have most updated `.favicon`

### PR DESCRIPTION
**Pull request by cdanwards**
Fixes #380 

## Creator

- [x] Build relevant tests if any apply
- [x] Ensure there are no new warnings
- [x] Upload an animated GIF showcasing the solution (if it applies)
- [x] Set a relevant reviewer

## Reviewer

- [x] Run `test` and make sure all tests pass
- [x] Review function and form on iOS
- [x] Review function and form on Android
- [x] Review function and form on Web
- [ ] Review new test logic

Adding the Newspring favicon and the spring splash screen

### CODE REVIEW

**Comment by cdanwards** [comment]
Still waiting for the assets for the splash screen to be delivered. Will update once they're received.

**Comment by cdanwards** [comment]
Just to note, I wasn't able to get the android splash screens to fill the screen unless I used 'cover'. Rich mentioned that we were trying to move away from using this, is that the case?

File: app-config.js [hunk 1]
```diff
@@ -21,20 +21,20 @@ module.exports = env => ({
...
-        backgroundColor: '#1c683e',
-        mdpi: 'src/assets/splash/android/320_470.png',
-        hdpi: 'src/assets/splash/android/480_640.png',
-        xhdpi: 'src/assets/splash/android/720_960.png',
-        xxhdpi: 'src/assets/splash/android/1080_1440.png',
+        mdpi: 'src/assets/splash/android/Spring2018_Splash_320x470.png',
+        hdpi: 'src/assets/splash/android/Spring2018_Splash_480x640.png',
+        xhdpi: 'src/assets/splash/android/Spring2018_Splash_720x960.png',
+        xxhdpi: 'src/assets/splash/android/Spring2018_Splash_1080x1440.png',
+        resizeMode: 'cover',
```
Comment (IsaacHardy) [inline app-config.js]: We shouldn't be using cover.

File: app-config.js [hunk 2]
```diff
@@ -21,20 +21,20 @@ module.exports = env => ({
       ],
       icon: 'src/assets/icons/ios/icon-1024x1024.png',
       splash: {
-        image: 'src/assets/splash/ios/768_1024.png',
-        backgroundColor: '#1c683e',
+        image: 'src/assets/splash/ios/splash-1242x2436.png',
+        tabletImage: 'src/assets/splash/ios/splash-2224x2732.png',
```
Comment (IsaacHardy) [inline app-config.js]: It seems this image isn't the size we need. I can resize it in Sketch to the size we need.

File: public/index.html [hunk 3]
```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="shortcut icon" href="/favicon.ico">
+    <link rel="shortcut icon" href="/dg0ddngxdz549.cloudfront.net/newspring/icons/favicon.ico?v=4">
```
Comment (burkeshartsis) [inline public/index.html]: I'm not seeing this change picked up in Netlify (default page icon). Locally I get a Differential logo.

Comment (IsaacHardy) [reply public/index.html]: The url is incorrect. Needs two `/`.

**Pull request merged**